app/src/main/java/com/android/calendar/month/MonthByWeekFragment.java

```diff
@@ -53,10 +53,7 @@
 import com.android.calendar.event.CreateEventDialogFragment;
 import com.android.calendarcommon2.Time;
 
-import java.util.ArrayList;
-import java.util.Calendar;
-import java.util.HashMap;
-import java.util.List;
+import java.util.*;
```

Comment:
please revert this as well. Could you please check for other occurrences yourself?